### ISSUE TITLE
A11y_VSCode_VersionControl_ScreenReader_ Screen reader does not announce about green blue red line indicator

### ISSUE THREAD

**Issue by prasadaps** [microsoft/vscode#77068]
**Environment Details:**
VSCode Version : 1.36.0
OS Version : Win10
Additional Details:
MAS Violated : MAS 1.3.1

**Repro Steps:**
1.Open Visual studio code editor
2.Using F1 key navigate to the palette, type "git branch" and select "Create Branch"
3. Enter the branch name in the edit field and hit enter. Observe the change in branch name at the bottom left corner
4. Publish branch to the remote Git repository by navigating to the bottom left pane, arriving at the cloud button next to the git branch and hit ENTER
5.Make changes to the code by refactoring, adding/deleting new lines ; using assistive technologies, observe if SRs will alert user the to blue/green/red indicators suggesting if code has been changed/new line added/line has been removed
6.Start Screen reader.
7. Observe screen reader announcement while navigating through each line in code editor.
**Actual:**
screen reader does not announce about new line is added on modified or deleted .

**Expected:**
screen reader should announce  about new line is added on modified or deleted .
Recommendations:
Refer below link which is repository of bug fixes code snippets:
https://microsoft.sharepoint.com/teams/msenable/mas/pages/browse-fixes.aspx
**User Impact**:
Keyboard and screen reader user will face difficulty to access this controls. 
**MAS Reference:**
MAS 2.1.1 - https://microsoft.sharepoint.com/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={54f28d1f-a2d1-4dcd-84e1-5c9b87e8aba4}
Attachment For Reference: 
[A11y_VSCode_VersionControl_ScreenReader_Screen reader does not announce about the green blue and red line indicator.pptx](https://github.com/microsoft/vscode/files/3376479/A11y_VSCode_VersionControl_ScreenReader_Screen.reader.does.not.announce.about.the.green.blue.and.red.line.indicator.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by isidorn** [comment]
This is simply not how this experience is designed.
User can press F7 and shift+F7 to go through the changes while in the diff view
And alt + F3 , alt + shit + F3 to go through the changes via zone widget - this nicely reads out all the changes in an acccessible way.
Closing as designed / will not support this
